Core/AppUrls.swift

```diff
@@ -23,14 +23,16 @@ import Foundation
 public struct AppUrls {
 
     private struct Url {
+
+        // You can change the base to use a subdomain for testing, but the unit tests will fail
```

Comment:
Can we update this so that the unit test don't fail? It would be good if a dev working off a test server could still run the tests. If you'e worried about them inadvertently checking in the wrong base url, we could have a single unit test which fails on that.